tests/functests/instancetype_test.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	vmReadyTimeout = 300 * time.Second
+	vmReadyTimeout = 900 * time.Second
```

Comment:
Has this been reported as an issue downstream somewhere?

I assume the issue is mostly with the huge size of the DVs being created rather than the time the OS takes to start and become ready?

I'd much prefer to either have a dedicated constant for the impacted Windows tests or have the ability to pass in a custom timeout into the test binary somehow for downstream to manage this. That might even allow us to lower the timeout for the other tests from the current 5 minutes that seems excessive.

Reply:
This came from my investigation when I tried the new images. 
I added new flag windows-ready-timeout by which the user can set higher timeout only for Windows VMs.